### DISCUSSION TITLE
How to set default terminal on Windows platform?

### DISCUSSION THREAD

**Discussion by nganun** [LazyVim/LazyVim#335] [Q&A] [answered]
Hello, you guys
When I open terminal in `LazyVim` on Windows platform, the default terminal is `CMD`.
How can I set the default terminal to `Powershell`, thanks!

**Comment by dpetka2001** [comment, marked as answer]
Haven't done this in Windows, so I don't have any personal experience, but found a topic on [reddit](https://www.reddit.com/r/neovim/comments/vpnhrl/how_do_i_make_neovim_use_powershell_for_external/) that seems to deal with the same problem. Check that out.

**Reply by nganun** [reply]
It works, thanks very much.
